srcDoc/release_notes.rst

```diff
@@ -62,6 +62,22 @@ following command-line parameters:
     --cpp-build
     --cs-build
 
+What's Fixed in Master
+~~~~~~~~~~~~~~~~~~~~~~
+
+Review Dynamic Data Send() and Receive() operations (#55)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*RTI Perftest* has optimize the `Send()` and `Received()` methods by:
+
+- Specifying the member_id in the `get_T()` and `set_T()` methods instead of
+  `DDS_DYNAMIC_DATA_MEMBER_ID_UNSPECIFIED`.
+- Creating a `std::map<std::string, int>` for the DDS_DynamicDataMemberIds.
```

Comment:
Maybe you don't need to include this, seems like a very specific detail for the implementation.